guides/tools/live-code-share.md

```diff
@@ -0,0 +1,26 @@
+# Live Code Sharing
+
+During the 4 hours pair programming interview you may need to use some tool to share the code from specific workspace. Or another use case is pair programming session with another teammate. There are a lot of different approaches to do that, however this one proposed by DVELP team.
```

Comment:
During the 4 hour pair programming interview we may need a tool to share code from a specific workspace or perhaps have a programming session with one of our teammates. From all the different approaches out there, at DVELP we propose the following: